api/src/lib/aws.ts

```diff
@@ -115,12 +116,11 @@ export async function s3PutSignedUrl(
   })
   return url
 }
+
 /**
  *  This function is a wrapper around the getSignedUrl function from the @aws-sdk/s3-request-presigner package.
  *  Exists to organize the imports and to make it easier to mock in tests.
  */
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This function's signature doesn't appear to have any unused vars, so removed this pragma.